packages/circuit-ui/components/TextArea/TextArea.tsx

```diff
@@ -13,13 +14,19 @@
  * limitations under the License.
  */
 
-import { forwardRef } from 'react';
+import { forwardRef, useRef } from 'react';
 import { css } from '@emotion/react';
 
 import Input from '../Input';
-import { InputProps } from '../Input/Input';
+import { HTMLCircuitInputElement, InputProps } from '../Input/Input';
+import { multiRefs } from '../../util/multiRefs';
 
-export type TextAreaProps = InputProps;
+import { useAutoExpand } from './useAutoExpand';
+
+export type TextAreaProps = Omit<InputProps, 'rows'> & {
+  rows?: InputProps['rows'] | 'auto';
+  minRows?: InputProps['rows'];
```

Comment:
Could you add type descriptions above their declarations? It helps a lot with intellisense 🙂